evap/evaluation/models.py

```diff
@@ -2059,17 +2061,17 @@ def send_to_users_in_evaluations(self, evaluations, recipient_groups, use_cc, re
 
     def send_to_user(self, user, subject_params, body_params, use_cc, additional_cc_users=(), request=None):
         if not user.email:
-            warning_message = (
-                f"{user.full_name_with_additional_info} has no email address defined. Could not send email."
+            message = _("{} has no email address defined. Could not send email.").format(
+                user.full_name_with_additional_info
             )
             # If this method is triggered by a cronjob changing evaluation states, the request is None.
             # In this case warnings should be sent to the admins via email (configured in the settings for logger.error).
             # If a request exists, the page is displayed in the browser and the message can be shown on the page (messages.warning).
             if request is not None:
-                logger.warning(warning_message)
-                messages.warning(request, _(warning_message))
+                logger.warning(message)
```

Comment:
It could be weird if we log translated messages here, because we might only find ~half of the messages if we ever decide to search/grep for the error message. Not a strong issue for me, but for consistency, it probably makes sense to strictly always log in english...